app/Http/Controllers/TodoController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TodoRequestValidation;
+use App\Services\TodoService;
+use App\Todo;
+
+class TodoController extends Controller
+{
+
+    private $service;
+
+    public function __construct(TodoService $service)
+    {
+        $this->service = $service;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $todos = $this->service->index();
+        return redirect('/')->with('success', compact('todos'));
```

Comment:
ne treba da radis redirect , tebi treba json response , kao sto sam zadatak kaze REST je u pitanju

dovoljno je da vratis samo $todos, a larevel ce ispod haube to znati kako da handle-a , da wrape-uje u JsonResponse i da stavi 200 status kod